#! /bin/sh /usr/share/dpatch/dpatch-run
## 50-FTBFS-gcc-4.0-fix.dpatch by Ondřej Surý <ondrej@sury.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix FTBFS when compiling with gcc-4.0, there is imapopts is used before declared

@DPATCH@
diff -urN cyrus22-imapd-2.2.12~/tools/config2header cyrus22-imapd-2.2.12/tools/config2header
--- cyrus22-imapd-2.2.12~/tools/config2header	2004-06-22 21:02:31.000000000 +0200
+++ cyrus22-imapd-2.2.12/tools/config2header	2005-06-22 07:56:17.180293688 +0200
@@ -214,7 +214,6 @@
 print HFILE <<EOF
   IMAPOPT_LAST
 };
-extern struct imapopt_s imapopts[];
 
 enum enum_value {
   IMAP_ENUM_ZERO = 0,
@@ -260,6 +259,8 @@
 
 print HFILE <<EOF
 };
+extern struct imapopt_s imapopts[];
+
 #endif /* INCLUDED_IMAPOPTIONS_H */
 EOF
     ;
